src/utils/helpers.js

```diff
@@ -73,3 +73,42 @@ export const flattenArray = arr =>
     (Array.isArray(item)
       ? [...acc, ...flattenArray(item)]
       : [...acc, item]), []).filter(item => !!item);
+
+// eslint-disable-next-line
```

Comment:
- 💅 please don't disable all eslint rules, but only the specific rule. Now it's not clear at all what is disabled and why is it needed and makes room for introducing more eslint violations.